packages/api/src/brevo/brevo.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from "@nestjs/common";
+
+import { ConfigModule } from "../config/config.module";
+import { BrevoContactsApiService } from "./brevo-contact-api.service";
+
+@Module({
+    imports: [ConfigModule],
+    providers: [BrevoContactsApiService],
+    exports: [BrevoContactsApiService],
+})
+export class BrevoModule {}
```

Comment:
why are there 2 BrevoModules?

Reply:
I dropped the 'comet' prefix in the src-folders root module.. I did not realize that the naming now exists 2 times. I guess this should now rather be called `BrevoApiModule` since this module contains all brevo logic. 
I will change the naming!

Reply:
Changed in 4d9b89ebf57c92cdc8217a5bb77b5a4b03419635